### ISSUE TITLE
BytesSerializer don't accept bytes-like objects

### ISSUE THREAD

**Issue by HeavenVolkoff** [eigenein/protobuf#46]
Hi,

The current `BytesSerializer` doesn't allow bytes-like objects to be passed, even though the `io.write` [natively supports it](https://docs.python.org/3/library/io.html#io.RawIOBase.write). Is there a reason for this?

https://github.com/eigenein/protobuf/blob/347979f87f4f562ffff0fad1440254923a288db8/pure_protobuf/serializers.py#L90-L108

I think changing the current `validate` function to something like:

``` python
def validate(self, value: Any): 
  if not isinstance(value, (bytes, bytearray, memoryview)):
    try:
      memoryview(value)
    except TypeError as exc:
      raise ValueError('a byte string is expected') from exc
```

Should allow bytes-like objects to work transparently. If that's fine with you, I could open a pull request.

**Comment by eigenein** [comment]
Hi,

There's no any specific reason, I've just missed that point :) But it indeed makes sense. I'll prepare a release today or tomorrow

Thanks for the hint!

**Comment by eigenein** [comment]
Fixed in `1.0.1`. The release is published on PyPI
